src/js/variant-widget/eva-variant-widget.js

```diff
@@ -1132,8 +1131,11 @@ function getProteinSubstitutionScore(consequenceTypes,so_array,source) {
         for (var j = 0; j < consequenceTypes[i].soTerms.length; j++) {
             if (consequenceTypes[i].soTerms[j].soName == _.first(so_array)) {
                 _.each(_.keys(consequenceTypes[i].proteinSubstitutionScores), function (key) {
-                    score = _.findWhere(this, {source: source}).score
-                    score_array.push(_.findWhere(this, {source: source}).score)
+                    var source_scores = _.findWhere(this, {source: source})
+                    if (source_scores !== undefined) {
+                        score = source_scores.score
+                        score_array.push(source_scores.score)
```

Comment:
Why not just **score_array.push(score)?**

Reply:
good point! Actually, the "score = source_scores.score" is not needed (score is initialized as "-" and then populated with the max score at the end of this function).